src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/PortableInfoboxPageObject.java

```diff
@@ -106,7 +110,7 @@ public String getCategoryLinkName() {
   }
 
   public String getUrlFromInternalLinkaAfterPageIsLoaded() {
```

Comment:
This method might be unnecessary and should be replaced with use of getCurrentURL from BasePageObject

why are you waiting for body ? 
Default webdriver behaviour is waiting for page to load
You don't have to wait for the body element unless you are using @UnstableLoadStrategy